src/contracts/core/AVSDirectory.sol

```diff
@@ -625,13 +628,15 @@ contract AVSDirectory is
         IStrategy strategy,
         OperatorSet calldata operatorSet
```

Comment:
shouldn't this function be edited to not be operatorSet specific

Reply:
too restrictive? allocations are limited to be 1 pending allocation per (op, opSet, Strategy).
Would be a bit inconsistent if deallos are (op, opSet) but its definitely a uglier check here vs just reading from pendingFreeMagnitude array

Reply:
didn't understand usage, my bad